user_templates/variables.cfg

```diff
@@ -85,8 +85,8 @@ variable_prime_line_height: 0.6 # mm, used for actual cross section computation
 variable_prime_line_length: 40 # length of the prime line on the bed (in mm)
 variable_prime_line_margin: 5  # distance of purge line from fl_size rectangle
 ## move into materials variable_prime_line_flowrate: 10 # mm3/s used for the prime line
-## move into materials variable_prime_line_pressure_length: 18 # distance to push filament to add  pressure into hotend (value near purge_retract_distanceif you purge before print )
-## move into materials variable_prime_line_purge_distance: 30 # length of filament to purge (in mm)
+## move into materials variable_prime_line_pressure_length: 18 # distance to push filament to add  pressure into hotend (value near standby_retract_lengthif you purge before print )
```

Comment:
Typo: missing space behind standby_retract_length

Reply:
Thanks, Fix in https://github.com/elpopo-eng/klippain-chocolate/pull/16/commits/10856e0a6a0c973478ee0d5502d4e9e105d9ebb6